crates/wasmi/src/store.rs

```diff
@@ -165,6 +183,20 @@ fn test_store_is_send_sync() {
     };
 }
 
+#[derive(Debug)]
+/// Argument to the callback set by [`Store::call_hook`] to indicate why the
+/// callback was invoked.
+pub enum CallHook {
```

Comment:
Not important but it itches me a bit to see `#[derive(Debug)]` above doc comments. 🙃 Rest of the codebase has doc comments first.